packages/lesswrong/components/ea-forum/MaintenanceBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { createStyles } from "@material-ui/core/styles";
+import { registerComponent, Components } from "../../lib/vulcan-lib";
+import { DatabasePublicSetting } from "../../lib/publicSettings";
+import { ExpandedDate } from "../common/FormatDate";
+
+export const maintenanceTime = new DatabasePublicSetting<string | null>('maintenanceBannerTime', null)
+const explanationText = new DatabasePublicSetting<string>('maintenanceBannerExplanationText', '')
+
+const styles = createStyles(
+  (theme: ThemeType): JssStyles => ({
+    root: {
+      padding: 20,
+      width: "100%",
+      fontFamily: theme.typography.postStyle.fontFamily,
+      fontSize: "1.5rem",
+      marginTop: "0.5em",
+      border: theme.palette.border.commentBorder,
+      borderWidth: 2,
+      borderRadius: 2,
+      borderColor: theme.palette.error.main,
```

Comment:
I'd say softer. Primary color.

Reply:
Maybe unless you're thinking we should show this only for the preceding 2 hours or something pretty short? And then use the featured banner the rest of the time?

Reply:
I was thinking we would use this in every case since making an image for the featured banner is a bit of a hassle. I've now made it so it turns red 2 hours before the maintenance time